packages/kibana/_dev/deploy/docker/docker-compose.yml

```
@@ -0,0 +1,48 @@
+version: "2.3"
+services:
+  elasticsearch:
+    image: "docker.elastic.co/elasticsearch/elasticsearch:${ELASTIC_VERSION}"
+    environment:
+      - "ES_JAVA_OPTS=-Xms1g -Xmx1g"
+      - "ELASTIC_PASSWORD=${ELASTIC_PASSWORD}"
+    ports:
+      - "127.0.0.1:9201:9200"
+    volumes:
+      - "./config/elasticsearch.yml:/usr/share/elasticsearch/config/elasticsearch.yml"
+    healthcheck:
+      test: curl -sfo /dev/null -u elastic:${ELASTIC_PASSWORD} localhost:9200 || exit 1
+      retries: 300
+      interval: 1s
+  setup:
+    image: "alpine/curl:latest"
+    environment:
+      - "ES_SERVICE_HOST=http://elasticsearch:9200"
+      - "KIBANA_PASSWORD=changeme"
```

Can do this to have it use the .env value

```suggestion
      - KIBANA_PASSWORD
```

---

good catch.